src/huntsman/pocs/core.py

```diff
@@ -8,15 +9,19 @@ def __init__(self, *args, **kwargs):
         self._dome_open_states = []
         super().__init__(*args, **kwargs)
 
-    def run(self, initial_next_state='starting', *args, **kwargs):
+    def run(self, initial_next_state='starting', skip_coarse_focus=False, *args, **kwargs):
         """ Override the default initial_next_state parameter from "ready" to "starting".
         This allows us to call pocs.run() as normal, without needing to specify the initial next
         state explicitly.
         Args:
             initial_next_state (str, optional): The first state the machine should move to from
                 the `sleeping` state, default `starting`.
+            skip_coarse_focus (bool, optional): If True, will skip the initial coarse focus.
+                Default False.
             *args, **kwargs: Parsed to POCS.run.
         """
+        if skip_coarse_focus:
+            self.observatory.last_coarse_focus_time = current_time()
```

Comment:
If you take suggestion up above for reversing logic then need:
```suggestion
        # Override last coarse focus time if not doing initial coarse focus.
        if initial_focus is False:
            self.observatory.last_coarse_focus_time = current_time()
```